pom.xml

```diff
@@ -56,6 +56,66 @@
         </repository>
     </repositories>
 
+    <build>
```

Comment:
@warunap - I still see 9 occurrences of throws Exception in java classes. Can you fix them as well.
e.g check - EmailSender.java